data/id.go

```diff
@@ -0,0 +1,70 @@
+// Autogenerated jute compiler
+// @generated from '/home/pmazzini/repos/zk/zookeeper.jute'
```

Comment:
having the full path here is not something we want. Can we do relative paths or not include the path?

Reply:
Yeah, I agree https://github.com/go-zookeeper/jute/issues/20